src/Commands/StartFrankenPhpCommand.php

```diff
@@ -0,0 +1,231 @@
+     *
+     * @return string
+     */
+    protected function buildMercureConfig()
+    {
+        if (! $mercure = (config('octane')['mercure'] ?? false)) {
+            return '';
+        }
+
+        $config = 'mercure {';
+
+        foreach ($mercure as $key => $value) {
+            if ($value === false) {
+                continue;
+            }
+
+            if ($value === true) {
+                $config .= "\n\t\t\t$key";
+
+                continue;
+            }
+
+            $config .= "\n\t\t\t$key $value";
+        }
+
+        if (! isset($mercure['demo']) &&
+            (strtoupper($this->option('log-level')) === 'DEBUG' || app()->environment('local'))
+        ) {
+            $config .= "\n\t\t\tdemo";
+        }
+
+        return "$config\n\t\t}";
+    }
+
+    /**
+     * Write the FrankenPHP server state file.
+     *
+     * @return void
+     */
+    protected function writeServerStateFile(
+        ServerStateFile $serverStateFile
+    ) {
+        $serverStateFile->writeState([
+            'appName' => config('app.name', 'Laravel'),
+            'host' => $this->getHost(),
+            'port' => $this->getPort(),
+            'workers' => $this->workerCount(),
+            'maxRequests' => $this->option('max-requests'),
+            'octaneConfig' => config('octane'),
+        ]);
+    }
+
+    /**
+     * Get the number of workers that should be started.
+     *
+     * @return int
+     */
+    protected function workerCount()
+    {
+        return $this->option('workers') === 'auto'
+            ? 0
+            : $this->option('workers');
+    }
+
+    /**
+     * Write the server process output to the console.
+     *
+     * @param  \Symfony\Component\Process\Process  $server
+     * @return void
+     */
+    protected function writeServerOutput($server)
+    {
+        [$_, $errorOutput] = $this->getServerOutput($server);
+
+        Str::of($errorOutput)
+            ->explode("\n")
+            ->filter()
+            ->each(function ($output) {
+                if (! is_array($debug = json_decode($output, true))) {
+                    return $this->info($output);
+                }
+
+                if ($debug['level'] == 'info'
+                    && isset($debug['request'])
+                    && $debug['msg'] == 'handled request') {
+                    [
+                        'duration' => $duration,
+                        'request' => [
+                            'method' => $method,
+                            'uri' => $url,
+                        ],
+                        'status' => $statusCode,
+                        'request' => $request,
+                    ] = $debug;
+
+                    return $this->requestInfo([
+                        'method' => $method,
+                        'url' => $url,
+                        'statusCode' => $statusCode,
+                        'duration' => (float) $duration * 1000,
+                    ]);
+                }
```

Comment:
This is missing the regular output (the startup/error logs are also json and are now ignored)

Reply:
Yeah - working on it.